src/tetris/scene/menuScene.ts

```diff
@@ -81,13 +82,21 @@ export default class MenuScene extends Phaser.Scene {
 	}
 
 	private _createButtons(): void {
-		const menuStartX: number = config.graphics.width / 2;
+		const buttonsXPosition: number = config.graphics.width / 2;
 		const menuStartY: number = config.graphics.height / 3;
 		const spacing: number = 20;
-		this._playButton = new TextButton(this, menuStartX, 0, "blue_button00.png", "blue_button01.png", "Join Matchmaking", this._joinMatchmaking.bind(this));
-		this._optionsButton = new TextButton(this, menuStartX, 0, "blue_button00.png", "blue_button01.png", "Leave Matchmaking", this._leaveMatchmaking.bind(this));
+
+		this._playButton = new TextButton(this, buttonsXPosition, 0, "blue_button00.png", "blue_button01.png", "Join Matchmaking", this._joinMatchmaking.bind(this));
+		this._optionsButton = new TextButton(this, buttonsXPosition, 0, "blue_button00.png", "blue_button01.png", "Leave Matchmaking", this._leaveMatchmaking.bind(this));
+
 		this._playButton.y = menuStartY;
 		this._optionsButton.y = menuStartY + this._playButton.height + spacing;
+
+		if(this._game.profiler) {
```

Comment:
It seems pretty weird, that the game can only be exited if the `Profiler` exists...
I guess this check should be performed at a different place.